controllers/convoController.js

```diff
@@ -111,25 +115,52 @@ const fetchConversationGet = async (req, res) => {
 
 // Send message
 const sendMessagePost = async (req, res) => {
-	// Removing leading and ending whitespace
-	const message = req.body.message.trim();
+	let messageText = req.body.message.trim();
 
 	try {
 		// If our message is empty then
-		if (!message) return res.sendStatus(406);
+		if (!messageText) return res.sendStatus(406);
 
 		// Otherwise, message is not empty to let's save it
 		const newMessage = new Message({
-			conversationID: req.params.id,
+			conversationID: req.body.conversationID,
 			sender: req.user._id,
-			content: message,
+			content: messageText,
 			timeCreated: new Date()
 		});
 
 		await newMessage.save();
+
+		/* 
+			Making a copy of the convo, deleting it, then saving
+			the copy so we can push it to the top of a user's list
+		*/
+
+		// Creating a copy
+		const copyOfConvo = await Conversation.findOne({
```

Comment:
oh is this copy of convo just meant to push it to the top of user's list? 

if so, instead of doing this, just keep track of when the last message was sent and update that field, then on user's list we sort it by that timestamp

Reply:
Wow, that's such a nice way of doing it. Going to implement this now, thanks Bonnie!

Reply:
Hey Bonnie, just implemented it! Let me know what you think.